packages/devtools_app/lib/src/screens/logging/logging_controller.dart

```diff
@@ -875,6 +875,29 @@ class LogData with SearchableDataMixin {
     }
   }
 
+  String get encodedDetails {
+    if (_encodedDetails != null) return _encodedDetails!;
+    if (details == null) return '';
+
+    // TODO(kenz): ensure this doesn't cause performance issues.
+    late String encoded;
```

Comment:
Does this even need to be `late`? It's initialized in both code paths below so I don't think `late` is necessary.